components/vf-button/vf-button.scss

```diff
@@ -55,12 +58,12 @@
     @include button-link--ghost;
   }
 
-  &.vf-text-button--secondary {
-    @include button-link--ghost($vf-link--color: set-color(vf-color--green), $vf-link--hover-color: set-color(vf-color--green--dark));
+  &.vf-button--secondary {
```

Comment:
Good change, and `vf-button-sm`, obviously will require some changes in consumers of the VF. We should bubble up to the changelog. _That awkward phase when things aren't stable but are in use._

Reply:
Thinking more on this. Yesterday I felt like the only action we could do would be to stick a note in the CHANGELOG.md, which would be good, but likely go unseen by most. Especially as it's not a major version change. 

I think this is a good chance to use the `vf-deprecated` approach. PR to the PR coming.

Reply:
I don't think it's deprecated in the same sense as we've been using 'deprecated'.

I agree a changeling in itself is not enough. I've spoken w/ David about it - and he's not using it anywhere yet.

Who else would be using it?

I have another idea - but I'll raise an issue for it - as it's something that's probably worth commenting on.

Reply:
The EBI devs might be using it on the EBI training project, but it won't be an issue there until they update their components. 

There might also be a page here or there that's using `vf-text-button--secondary`

The trickier one is the [data protection banner](https://dev.beta.embl.org/api/v1/pattern.html?filter-content-type=article&filter-id=581&pattern=node-body&source=contenthub) that's served from the contentHub. Once the JS runs it renders a `vf-button vf-text-button--secondary` — and after this PR it will doe a `vf-button vf-button--secondary`.

And if a site is using an older version of vf-button the button will look wrong.

That particular issue could be worked around by making vf-banner also use the old class, but that feels messy? and would be a less general fix.

Reply:
how `vf-text-button--secondary` got into this I cannot remember.

But it's really inconsistent, don't you think? 

From searching the codebase, it feels like (to me) a find/replace that was done - and this got missed as a bad outcome.

Reply:
oh yeah, for sure it must have been some find-replace gone mad. will be very happy to see it go — did you see the PR I made, that keeps your change and keeps `vf-text-button--secondary` for deprecated support.

Reply:
yep, left for you to merge - what needs to be done w/ this PR -- close it?

Reply:
Cool. Merged my PR into this one, and now this one gets merged into dev.